packages/api/cms-api/src/common/sorting/sort.args.ts

```diff
@@ -14,3 +14,15 @@ export class SortArgs {
     @IsEnum(SortDirection)
     sortDirection?: SortDirection = SortDirection.ASC;
 }
+
+@InputType()
+export class SortInput {
```

Comment:
Could you remove this SortInput by using an IntersectionType for the DamItemPositionArgs?